.vscode/tasks.json

```diff
@@ -16,6 +16,132 @@
         "isDefault": true
       }
     },
+    {
+      "label": "Build neovim extension only",
+      "dependsOn": [
+        "neovim start",
+        "ESBuild-neovim-registry",
+        "ESBuild-cursorless-neovim"
+      ],
+      "group": "build"
+    },
+    {
+      "label": "Build neovim extension tests only",
+      "dependsOn": [
+        "neovim test start",
+        "ESBuild-neovim-registry",
+        "ESBuild-cursorless-neovim",
+        "Build test harness"
+      ],
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-neovim-registry",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/neovim-registry",
+      "dependsOn": [],
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-cursorless-neovim",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/cursorless-neovim",
+      "dependsOn": [
+        "ESBuild-common",
+        "ESBuild-cursorless-engine",
+        "ESBuild-neovim-common"
+      ],
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-common",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/common",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-cursorless-engine",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/cursorless-engine",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "ESBuild-neovim-common",
+      "type": "npm",
+      "script": "compile",
+      "path": "packages/neovim-common",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "group": "build"
+    },
+    {
+      "label": "neovim logs",
+      "type": "shell",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "(New-Object -ComObject WScript.Shell).Run(\"\"\"powershell Get-Content '${workspaceFolder}/packages/cursorless-neovim/out/nvim_node.log' -wait -tail 30\"\"\", 1, $false)"
+        ]
+      },
+      "group": "build"
+    },
+    {
+      "label": "neovim start",
+      "type": "process",
+      "windows": {
+        "command": "powershell",
+        "args": [
+          "(New-Object -ComObject WScript.Shell).Run(\"\"\"nvim\"\"\", 1, $false)"
+        ]
+      },
+      "group": "build",
+      "options": {
+        "env": {
+          "CURSORLESS_REPO_ROOT": "${workspaceFolder}",
+          "NVIM_NODE_HOST_DEBUG": "1",
+          "NVIM_NODE_LOG_FILE": "${workspaceFolder}/packages/cursorless-neovim/out/nvim_node.log",
+          "NVIM_NODE_LOG_LEVEL": "info",
+          "CURSORLESS_MODE": "development"
+        }
+      }
+    },
+    {
+      "label": "neovim test start",
+      "type": "process",
+      "windows": {
```

Comment:
looks like you don't have any configuration for mac? would be nice if I could get this thing running locally

Reply:
At the moment I've only tested it on Windows so it would need to be tested on mac and Linux. Happy to help you with configuring the debug environment locally if you want in a future meeting. You may want to install the production cursorless.nvim first to solve all the other dependencies though.